testsuite/modern-testsuite/src/test/java/org/sonatype/nexus/testsuite/raw/RawITSupport.java

```diff
@@ -35,6 +36,7 @@
 /**
  * Support class for raw ITs.
  */
+@Category(RawTestGroup.class)
```

Comment:
https://github.com/lucaspouzac/contiperf/releases/tag/release-2.4.3 is the release with the fix for junit 4.12